lighthouse-core/audits/accessibility/duplicate-id-active.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Ensures every `id` attribute value of active, focusable elements is unique.
+ * See base class in axe-audit.js for audit() implementation.
+ */
+
+const AxeAudit = require('./axe-audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that checks if there are any duplicate id HTML attributes on active, focusable elements. This title is descriptive of the successful state and is shown to users when no user action is required. */
```

Comment:
meta: I'm not sure why we need to bother with either of these when we already have a `duplicate-id` that applies to the entire page already...

Reply:
> meta: I'm not sure why we need to bother with either of these when we already have a `duplicate-id` that applies to the entire page already...

well, interestingly, I was going to leave them out thinking the same thing, but somehow failures from these don't show up in `duplicate-id` (see for instance the smoke test where `duplicate-id` didn't have to add new items).

I'm not sure why this happens, there's nothing about that in the `duplicate-id` test definition. Maybe it's something axe does if you have overlapping tests somehow? It's a big mystery (we should probably figure it out before landing :)